gen.go

```diff
@@ -0,0 +1,74 @@
+// +build ignore
+
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"text/template"
+	"time"
+)
+
+func main() {
+	const url = "https://mkcert.org/generate/"
+	resp, err := http.Get(url)
+	if err != nil {
+		log.Fatal(err)
+	}
+	if resp.StatusCode != 200 {
+		log.Fatal("expected 200, got", resp.StatusCode)
+	}
+	defer resp.Body.Close()
+
+	bundle, err := ioutil.ReadAll(resp.Body)
```

Comment:
Is there any way we could parse the PEM file here and represent it in such a way that no parsing is required at runtime? It would be nice if `CACerts()` didn't have the potential to return an error.

Reply:
That's interesting. I assume so? But I'd have to dig into the source. We'd need to bypass the `pool.AppendCertsFromPEM` and just assign something directly.

Reply:
Yeah, so just need to build up a Certificate object, then use `AddCert` instead: https://golang.org/src/crypto/x509/cert_pool.go?s=3476:3540#L129

Reply:
Cool. Now would probably be a good time to make this change, since the versioning scheme is already being changed. Let me know if you want a hand with the generator code for this.

Reply:
Yup, agreed. I'm not sure when I'll get time, but I'm 99% confident we can massage these directly into Certificate objects at generation time.

Reply:
I'm going to merge this in now as-is since behavior isn't changing, and this just gets us updated certs. Without tags, likely things will pull master anyways, which is fine.

I'm going to dig into this now and see if it's worth generating out full Certificate objects.